index.js

```diff
@@ -43,14 +45,56 @@ var PhantomJSBrowser = function(baseBrowserDecorator, config, args) {
       }));
     }
 
-    var captureCode = 'var page = require("webpage").create();\n' +
+    var captureCode;
+    if (args.debug) {
+      flags = flags.concat('--remote-debugger-port=9000');
+      flags = flags.concat('--remote-debugger-autorun=yes');
+
+      captureCode =
+        "var system = require('system'),\n" +
+        "fs = require('fs'),\n" +
+        "webpage = require('webpage');\n" +
+        "\n" +
+        "(function (phantom) {\n" +
+        "  var page = webpage.create();\n" +
+        "\n" +
+        "  function debugPage() {\n" +
+        "    console.log('Launch the debugger page at http://localhost:9000/webkit/inspector/inspector.html?page=2');\n" +
+        "\n" +
+        "    var debuggerWait = 15000;\n" +
+        "    console.log('Waiting ' + (debuggerWait / 1000) + ' seconds for debugger page to launch...');\n" +
+        "\n" +
+        "    var launchPage = function () {\n" +
+        "      console.log('Launching page " + url + "...');\n" +
+        "      page.open('" + url + "');\n" +
+        "    };\n" +
+        "\n" +
+        "    setTimeout(launchPage, 15000);\n" +
+        "  }\n" +
+        "  debugPage();\n" +
+        "}(phantom));\n" +
+        "\n";
```

Comment:
I'm not a big fan of having that much code in quotes, how about you put this into a separate file and then do something like this:

```js
// capture.template
..
page.open('<%= url %>');
..

// ---
fs.readFile('./capture.templates.js', function (err, file) {
  captureCode = _.template(file.toString(), {url: url})
})
```